README.md

```diff
@@ -1,24 +1,28 @@
 ## VSCode Projects Runner
-This plugin use the Project Manager plugin from VSCode to read the projects. 
+
+This plugin use the Project Manager plugin from VSCode to read the projects.
 
 ## Required Dependencies
 
-Debian/Ubuntu:  
+Debian/Ubuntu:
```

Comment:
Is there any reason you removed the two trailing spaces? The line break was intentional or is there any reason against it?

Reply:
It was made by assident. Will fix it todat

Reply:
Ok. It was made by my formatter in vscode. Will fix it

Reply:
@alex1701c Fixed